databuilder/query_engines/sqlite.py

```diff
@@ -84,11 +84,21 @@ def get_sql_not(self, node):
 
     @get_sql.register(Function.And)
     def get_sql_and(self, node):
-        return operators.and_(self.get_sql(node.lhs), self.get_sql(node.rhs))
+        # SQLAlchemy doesn't provide reverse bitwise operations, so `True & Column()` raises a `TypeError`.
+        # See https://github.com/sqlalchemy/sqlalchemy/issues/5846.
+        try:
+            return self.get_sql(node.lhs) & self.get_sql(node.rhs)
```

Comment:
How come the switch back to operator overloading here, rather than using `operators.and_`?

Reply:
I think this fix pre-dates your switching to functions and I missed it in the merge. Fixed now.